tsconfig.json

```diff
@@ -12,11 +12,16 @@
     "outDir": "dist",
     "strict": true,
     "esModuleInterop": true,
-    "declaration": true
+    "declaration": true,
+    "resolveJsonModule": true
   },
   "exclude": [
```

Comment:
Just realised, that maybe we should also include `"src/**/__tests__/**/*.ts"` here so that the `__tests__` folder is excluded from the production build of the package